deno/payloads/v10/channel.ts

```diff
@@ -197,7 +197,7 @@ export interface APIThreadChannel
 	 */
 	thread_metadata?: APIThreadMetadata;
 	/**
-	 * The approximate message count of the thread, does not count above 50 even if there are more messages
+	 * Count of messages (not including starter message and deleted messages) in a thread, stops counting at 100000 (if the thread is created before July 1, 2022, it stops at 50)
```

Comment:
You should consider splitting these into multiple sentences because it’s a bit confusing. Also you should clarify that this *does* decrement on message deletion